src/components/IOUConfirmationList.js

```diff
@@ -40,6 +40,9 @@ const propTypes = {
     /** Should we request a single or multiple participant selection from user */
     hasMultipleParticipants: PropTypes.bool.isRequired,
 
+    /** Can the set of participants be adjusted? */
+    canDeselectParticipants: PropTypes.bool,
```

Comment:
**thought:** `canDeselectParticipants` maybe counts as a negated variable? Would `canModifyParticipants` or `hasModifiableParticipants` be clearer?

**thought:** the comment that asks if it can be "adjusted" could be clearer - I asked myself "adjust how?" Does this mean that some participant is shown as "selectable", but we cannot "deselect" them? If so, I'm curious to understand this UX pattern. If they can't be de-selected why show them as "selected"?

Reply:
> canDeselectParticipants maybe counts as a negated variable? Would canModifyParticipants or hasModifiableParticipants be clearer?

Good point, I'll make this `canModifyParticipants`